### PR TITLE
Add uppercase H in GitHub

### PR DESCRIPTION
**Pull request by AngelOnFira**
This is pretty pedantic, but I've become quite eagle-eyed on noticing the lowercase `h` on Github 😅 this is just replacing anything that matched against the case sensitive `Github`, so it should only change visible items.

### CODE REVIEW

File: content/news/002/index.md [hunk 1]
```diff
@@ -527,7 +527,7 @@ but I'm too tired to this atm and it will be rendered correctly anyway.
 -->
 
 [Alexandru Ene]: https://twitter.com/_AlexEne_
-["Github Actions CI with Rust and SDL2"]: https://alexene.dev/2019/09/04/Github-actions-CI-rust-SDL2.html
+["GitHub Actions CI with Rust and SDL2"]: https://alexene.dev/2019/09/04/GitHub-actions-CI-rust-SDL2.html
```
Comment (janhohenheim) [inline content/news/002/index.md:530]: This breaks the hyperlink ;)
```suggestion
["GitHub Actions CI with Rust and SDL2"]: https://alexene.dev/2019/09/04/Github-actions-CI-rust-SDL2.html
```

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: content/news/002/index.md (hunk 1 above)
Comment (AngelOnFira) [reply content/news/002/index.md:530]: So it does 😄 thanks for the catch!

**Review by janhohenheim — APPROVED** [review]
LGTM

**Review by janhohenheim — APPROVED** [review]
LGTM

**Pull request merged**